api/api_lib.php

```diff
@@ -70,7 +75,7 @@ function createAssignment($assignment_params){
     return $assignment;
 }
 
-function setup_radio_question($assignment, $rubric_params){
+function create_radio_question($assignment, $rubric_params){
```

Comment:
my thought was that it only really works on radio questions right now, review is more of a general term as there are other types of reviews. maybe setup_review_radio_question ?